src/extended_json.ts

```diff
@@ -234,6 +234,13 @@ function serializeValue(value: any, options: EJSONSerializeOptions): any {
     return { $numberDouble: Object.is(value, -0) ? '-0.0' : value.toString() };
   }
 
+  if (typeof value === 'bigint') {
+    if (!options.relaxed) {
+      return { $numberLong: BigInt.asIntN(64, value).toString() };
```

Comment:
This is done to truncate the BigInt to 64bits to mirror the new behaviour of `BSON.serialize` for BigInt